diff --git a/makefile.in b/makefile.in
index 69b4d22..1ad93fd 100644
--- a/makefile.in
+++ b/makefile.in
@@ -47,6 +47,9 @@ mobical: mobical.o libcadical.a makefile
 libcadical.a: $(OBJ) makefile
 	ar rc $@ $(OBJ)

+libcadical.so: $(OBJ) makefile
+	$(COMPILE) -shared -o $@ $(OBJ)
+
 #--------------------------------------------------------------------------#

 # Note that 'build.hpp' is generated and resides in the build directory.
